src/TestSuite/Constraint/Email/MailContains.php

```diff
@@ -52,6 +52,32 @@ public function matches($other): bool
         return false;
     }
 
+    /**
+     * @return string
+     */
+    private function getTypeMethod(): string
+    {
+        return 'getBody' . ($this->type ? ucfirst($this->type) : 'String');
+    }
+
+    /**
+     * Returns the type-dependent strings of all messages
+     * 
+     * @return string
+     */
+    protected function getAssertedMessages(): string
+    {
+        $messageMembers = [];
+        $messages = $this->getMessages();
+        foreach ($messages as $message) {
+            $method = $this->getTypeMethod();
+            $messageMembers[] = $message->$method();
+        }
+        $result = join(LF, $messageMembers);
+
+        return LF . 'was: ' . substr($result, 0, 1000);
```

Comment:
mb_ function is important here
And usually we use PHP_EOL.

Reply:
ok I just changed it